lib/embed/stacks_media_stream.rb

```diff
@@ -47,7 +47,8 @@ def streaming_file_prefix
       if video?
         'mp4'
       elsif audio?
-        'mp3'
+        return 'mp3' if file_extension == 'mp3'
+        return 'mp4' if file_extension == 'm4a'
```

Comment:
i hit approve, but this is also a reasonable point.  agreed on leaving it up to @tingulfsen and maybe waiting for another instance in case an obvious pattern emerges (though config hash seems like a sensible idea).